test/native/setup.js

```diff
@@ -19,24 +19,37 @@ global.ReanimatedDataMock = {
 	now: () => 0,
 };
 
-RNNativeModules.UIManager = RNNativeModules.UIManager || {};
-RNNativeModules.UIManager.RCTView = RNNativeModules.UIManager.RCTView || {};
-RNNativeModules.RNGestureHandlerModule =
-	RNNativeModules.RNGestureHandlerModule || {
-		State: {
-			BEGAN: 'BEGAN',
-			FAILED: 'FAILED',
-			ACTIVE: 'ACTIVE',
-			END: 'END',
-		},
-		attachGestureHandler: jest.fn(),
-		createGestureHandler: jest.fn(),
-		dropGestureHandler: jest.fn(),
-		updateGestureHandler: jest.fn(),
+jest.mock( 'react-native', () => {
```

Comment:
I noticed when mocking the WebView native module that in some tests the mock was restored, leading to failures.
Mocking the native modules this way should prevent this issue.